specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/apigateway.json

```diff
@@ -666,6 +666,29 @@
           "type": "string",
           "description": "ETag of the resource.",
           "readOnly": true
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
```

Comment:
`Kind` cannot be a readonly property. The client need to specify what `kind` of Gateway or Workspace they are creating. `Readonly` is reserved for properties which only the server can set

Reply:
Agreed, I have missed that one

Reply:
Removed.